contract-testing/pact-messaging/provider/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id("org.springframework.boot")
+    id("io.spring.dependency-management")
+
+    kotlin("jvm")
+    kotlin("plugin.spring")
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-web")
```

Comment:
just `org.springframework.boot:spring-boot-starter` should be enough

Reply:
done